### DISCUSSION TITLE
armbian无法从U盘启动

### DISCUSSION THREAD

**Discussion by 3611895075** [ophub/amlogic-s9xxx-armbian#1553] [Amlogic]
在cm311-1a上用U盘刷debian12到emmc。其他系统不能从U盘启动，除了debian12

**Comment by 3611895075** [comment]
替换了boot文件夹里的boot.cmd和boot.scr后，就能从U盘启动，这是什么玄学？

**Reply by btc-wang** [reply]
用什么文件替换的

**Reply by 3611895075** [reply]
用原U盘的boot文件。我试过了，只能启动o大的debian和ubuntu系统
